modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -893,6 +953,28 @@ private ObjectRelationship _addObjectRelationshipSystemObjectDefinition()
 			ObjectRelationshipConstants.TYPE_ONE_TO_MANY, null);
 	}
 
+	private ObjectRelationship _createObjectRelationshipEdge(
```

Comment:
Can you show an example of methods in test classes that use the prefix create?

What do you think about `_addObjectRelationshipEdge()` ?